invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides

```diff
@@ -126,3 +126,14 @@
 }
 
 
+.item {
+  .right-bottom-content{
+      @media all and (min-width: @largestMobileScreen) {
+        float: right;
+      }
+      @media all and (max-width: @largestMobileScreen) {
+        display: block;
+        margin-bottom: 1em;
+      }
+  }
+}
```

Comment:
I would need to duplicate a big amount of JSX, to avoid using this css. Maybe I could move it to a separate component but I would like if I could get any advice in the review.

Reply:
you can use mobile only, we can see together tomorrow how to implement it, or you can check how we deal with it in ILS